### PR TITLE
install agent : add  boolean parameter for ignoreSysPrerequisite, bec…

### PR DESCRIPTION
**Pull request by pmpetit**
…ause agent 12.1.0.3 is not certified on redhat 7, but agent 12.1.0.3 runs on redhat 7.*

### CODE REVIEW

**Comment by biemond** [comment]
nice, can you resolve the conflict, after that  I will merge it.

thanks again

**Comment by pmpetit** [comment]
Done,
Do you have an idea about the moment you will release  this version ?



*Pierre-Marie PETIT*
SI Engineer
Tech & Talents - B'TWIN

*+33 6 48 26 78 93Check our new
website: https://techandtalents.subsidia.org/
<https://techandtalents.subsidia.org/>*

File: manifests/installem_agent.pp [hunk 1]
```diff
@@ -242,10 +243,15 @@
                       Oradb::Utils::Dborainst["em agent orainst ${version}"],],
       }
 
+      $param_ignore_prereq=''
+      if ( $ignore_sys_prerequisite ) {
+        $param_ignore_prereq='-ignorePrereqs'
```
Comment (alexjfisher) [inline manifests/installem_agent.pp:248]: Does this actually work?
https://puppet.com/docs/puppet/6.0/lang_variables.html#no-reassignment

Comment (pmpetit) [reply manifests/installem_agent.pp:248]: oops, i'm having doubt......

Comment (pmpetit) [reply manifests/installem_agent.pp:248]: should have use else instead....

Comment (biemond) [reply manifests/installem_agent.pp:248]: indeed, good find

Comment (pmpetit) [reply manifests/installem_agent.pp:248]: sorry about that...

**Pull request merged**